### PR TITLE
[ch7300] Alert Dashboard Filters

### PR DESCRIPTION
**Pull request by domdinicola**
[ch7300]

### CODE REVIEW

**Comment by domdinicola** [comment]
@ntrncic let's talk about the first problem
@drzackyll / @marko911 can you give me the criteria to address the disabled filters?

**Comment by drzackyll** [comment]
@domdinicola i was originally given the criteria for the other filters by evrim. she told me the names of the two disabled filters, but didn't give me criteria. i'll check with her.

File: src/etools/applications/partners/views/dashboards.py [hunk 1]
```diff
@@ -83,28 +125,7 @@ def append_final_partnership_review(self, serializer):
         return serializer
 
     def append_last_pv_date(self, serializer):
-        delta = ExpressionWrapper(
-            datetime.now() - F('last_pv_date'),
-            output_field=DurationField()
-        )
-        qs = Intervention.objects.exclude(
-            status=Intervention.DRAFT
```
Comment (reinbach) [inline src/etools/applications/partners/views/dashboards.py]: The exclusion of Intervention being in `DRAFT` status does not appear to have been carried over, is that intentional?
Unless I'm missing something

Comment (domdinicola) [reply src/etools/applications/partners/views/dashboards.py]: mmm it should (new version: L67)

**Comment by robertavram** [comment]
why is this on hold?

**Comment by domdinicola** [comment]
I think we can close this.. we said we will filter by pk

**Pull request closed without merging**